python/code/test/test_manifold_discriminant.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from classificationmethods import ManifoldDiscrimantAnalysisParafacTucker as pt
+import numpy as np
+from numpy import random as random
+import scipy.stats as stats
+
+"""
+    Unit tests for python manifold discriminant analysis. 
+"""
+
+
+def generate_data(classes, length_of_data_set, shape=None):
```

Comment:
Would it be okay to add a few comments to this? I am not sure I understand the purpose of end_padding, and why the number of copies is calculated like it is.